### PR TITLE
[git] Add non-production files to .gitattributes

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -10,5 +10,12 @@
 *.xml text eol=lf
 *.dist text eol=lf
 *.neon text eol=lf
```
Comment (mimmi20) [inline .gitattributes]: Please set this to `export-ignore` too.

Comment (Slamdunk) [reply .gitattributes]: I don't like very much export-ignoring asterisks, there is no control on what in the future could be ignored.
phpstan.neon was already added few line below.

Comment (mimmi20) [reply .gitattributes]: Then remove this line please. phpstan.neon is the only *.neon file at the moment.

**Comment by Slamdunk** [comment]
Done

**Comment by asgrim** [comment]
Thank you @Slamdunk! :+1: :smile:

**Pull request merged**